### ISSUE TITLE
duplicated softmax functions are called in a single model

### ISSUE THREAD

**Issue by shengyehchen** [DSKSD/cs-224n-Pytorch#2]
**torch.nn.CrossEntropyLoss()** combines _LogSoftMax_ and _NLLLoss_ in one single class.
Thus, there should be only **_"return out"_** instead of return **_"F.log_softmax(out)"_** in the _forward()_ function of some classification models.
